### DISCUSSION TITLE
MintJoomla Cobalt API

### DISCUSSION THREAD

**Discussion by kiranetph** [JoomCoder-com/JoomCCK#235] [Q&A]
Was the API for creating and updating records successfully implemented, or do we have a different way to do it
?
--> CobaltApi::createRecord($data, $section_id, $type_id, $fields, $categories, $tags)
-->CobaltApi::updateRecord($redord_id, $data, $fields, $categories, $tags);

It was never fully working before.

**Comment by andiamine** [comment]
Hi the new call is tru class JoomcckApi

--> JoomcckApi::createRecord($data, $section_id, $type_id, $fields, $categories, $tags)
-->JoomcckApi::updateRecord($redord_id, $data, $fields, $categories, $tags);


test it , if you find any issues let us know
